README.md

```diff
@@ -1,33 +1,27 @@
-# taskflow
+# goyek
 
 > Create build pipelines in Go
 
-[![Go Reference](https://pkg.go.dev/badge/github.com/pellared/taskflow.svg)](https://pkg.go.dev/github.com/pellared/taskflow)
+[![Go Reference](https://pkg.go.dev/badge/github.com/goyek/goyek.svg)](https://pkg.go.dev/github.com/goyek/goyek)
 [![Keep a Changelog](https://img.shields.io/badge/changelog-Keep%20a%20Changelog-%23E05735)](CHANGELOG.md)
-[![GitHub Release](https://img.shields.io/github/v/release/pellared/taskflow)](https://github.com/pellared/taskflow/releases)
-[![go.mod](https://img.shields.io/github/go-mod/go-version/pellared/taskflow)](go.mod)
-[![LICENSE](https://img.shields.io/github/license/pellared/taskflow)](LICENSE)
-[![Build Status](https://img.shields.io/github/workflow/status/pellared/taskflow/build)](https://github.com/pellared/taskflow/actions?query=workflow%3Abuild+branch%3Amain)
-[![Go Report Card](https://goreportcard.com/badge/github.com/pellared/taskflow)](https://goreportcard.com/report/github.com/pellared/taskflow)
-[![codecov](https://codecov.io/gh/pellared/taskflow/branch/main/graph/badge.svg)](https://codecov.io/gh/pellared/taskflow)
+[![GitHub Release](https://img.shields.io/github/v/release/goyek/goyek)](https://github.com/goyek/goyek/releases)
+[![go.mod](https://img.shields.io/github/go-mod/go-version/goyek/goyek)](go.mod)
+[![LICENSE](https://img.shields.io/github/license/goyek/goyek)](LICENSE)
+
+[![Build Status](https://img.shields.io/github/workflow/status/goyek/goyek/build)](https://github.com/goyek/goyek/actions?query=workflow%3Abuild+branch%3Amain)
+[![Go Report Card](https://goreportcard.com/badge/github.com/goyek/goyek)](https://goreportcard.com/report/github.com/goyek/goyek)
+[![codecov](https://codecov.io/gh/goyek/goyek/branch/main/graph/badge.svg)](https://codecov.io/gh/goyek/goyek)
 [![Mentioned in Awesome Go](https://awesome.re/mentioned-badge.svg)](https://github.com/avelino/awesome-go)
```

Comment:
Informal: Where is the task list for all the side-tasks? awesome-go might also need an update to the new repo.

Reply:
I've created #47 for this.